app/init/push_notifications.ts

```diff
@@ -134,7 +134,7 @@ class PushNotifications {
             const isTabletDevice = await isTablet();
             const activeServerUrl = await getActiveServerUrl();
             const displayName = await queryServerName(DatabaseManager.appDatabase!.database, serverUrl);
```

Comment:
did we miss this one?

Reply:
I did not change any of the queries on the App database on purpose. Not so familiar with the code, and big enough change already. Do you mind if we postpone for a new PR?